templates/artifact_store.py

```diff
@@ -0,0 +1,69 @@
+import io
+import logging
+import sys
+
+from pathlib import Path
+
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+
+
+class ArtifactStore:
```

Comment:
@chelseatroy would you kindly file a ticket for adding test coverage to this?

Reply:
Will do!